src/test/resources/alma-fix/990223521400206441.json

```diff
@@ -238,23 +238,23 @@
     "serialNumber" : "1167639-10",
     "currentLocation" : "I5002 / IS",
     "heldBy" : {
-      "id" : "http://lobid.org/organisations/DE-Hag4#!",
-      "isil" : "DE-Hag4",
+      "id" : "http://lobid.org/organisations/DE-Hag4-2#!",
+      "isil" : "DE-Hag4-2",
       "label" : "lobid Organisation"
     },
-    "id" : "http://lobid.org/items/990223521400206441:DE-Hag4:Tgm34 4:3#!"
+    "id" : "http://lobid.org/items/990223521400206441:DE-Hag4-2:Tgm34 4:3#!"
```

Comment:
This is odd since the next id is identical and we have to think about how to make the URI unique.